Automattic/wp-calypso
# Log in: make form divider text transparent

## Proposed Changes

This makes the "OR" divider text transparent, so it blends in with whatever background. It also uses CSS adjacent selectors to merge any two cards around the divider and allow the divider to do the separation, not the cards themselves. 

Fixes https://github.com/Automattic/wp-calypso/issues/84548

## Testing Instructions

1. Please follow the testing steps here: https://github.com/Automattic/wp-calypso/pull/83965
2. Please make sure [the issue](https://github.com/Automattic/wp-calypso/issues/84548) is fixed.
